src/EnergyPlus/SolarShading.cc

```diff
@@ -3691,12 +3691,12 @@ namespace SolarShading {
 			WithShdgHoriz = 0.;
 			WoShdgHoriz = 0.;
 
-			for ( int IPhi = 1; IPhi <= NPhi; ++IPhi ) { // Loop over patch altitude values
-				SUNCOS( 3 ) = SinPhi[ IPhi ];
+			for ( int IPhi = 0; IPhi < NPhi; ++IPhi ) { // Loop over patch altitude values
+				SUNCOS( 3 ) = SinPhi[IPhi];
```

Comment:
I prefer appropriate padding.  The convention EnergyPlus adopted from the translation is to pad everything.  Perhaps we can make a huge push to clear out PRs after the release and make a big move to finalize our style and get a style checker to verify pushed code passes.